ci/docker/runtime_functions.sh

```diff
@@ -1307,13 +1307,14 @@ cd_pypi_publish() {
 
 cd_s3_publish() {
     set -ex
-    pip3 install --user awscli
+    pip3 install --upgrade --user awscli
     filepath=$(readlink -f wheel_build/dist/*.whl)
     filename=$(basename $filepath)
     variant=$(echo $filename | cut -d'-' -f1 | cut -d'_' -f2 -s)
     if [ -z "${variant}" ]; then
         variant="cpu"
     fi
+    export PATH=/usr/local/bin:$PATH
```

Comment:
Do you mean that `/usr/local/bin` wasn't on the PATH before? Do you know why it worked at all before?

Reply:
We recently changed the ami used by restricted-mxnetlinux-gpu to update the nvidia driver

Reply:
Do you think if there is a better place to set PATH?

Reply:
@leezu

Reply:
It's fine to add it here, I'm just trying to understand the reasoning. Can you explain how it's related to AMI? Wouldn't `cd_s3_publish` run in a docker container and be independent from the AMI?